lab/fasttrack.md

```diff
@@ -48,7 +58,7 @@ Once the deployment is finished, visit your site by clicking the link.
 
 ## Teardown
 
-You can tear down your application by running the following command.
+You can tear down your application by running the following command:
```

Comment:
Part of me thinks we should make this more obvious, with more warning emojis or something, so that people don't leave it running. (I say this as I look at my personal Azure Portal today and see I've incurred more accidental costs for things I left running!)

Reply:
@pamelafox - how's this?

```suggestion

> **Warning: **
> 
> **Teardown your project when finished with the lab to avoid recurring charges.**
> Though we've made an effort to utilize the cheapest and whenever possible free resources, leaving these resources running may accrue charges. Please completely de-provision your resources once no longer needed.

You can tear down your application by running the following command:
```